### ISSUE TITLE
Previous Ticket was closed/solved. Issue persists after update.

### ISSUE THREAD

**Issue by Debugga-nb** [Nnoggie/MythicDungeonTools#520]
Is this an issue with one of my existing routes? do routes before a certain date not work?
Same issue, error on shrink to MDT-Guide mode.

GetCurrentPreset: ...e/AddOns/MythicDungeonTools/Modules/PullOutlines.lua:8: attempt to compare two nil values
DrawHullFontString: ...e/AddOns/MythicDungeonTools/Modules/PullOutlines.lua:66: attempt to perform arithmetic on field '?' (a nil value)

10/08/23 11:01:14
MDT: 4.0.4.24
Client: 100105 enUS
Character: Corønus-WyrmrestAccord (US)
Out of combat
Zone: Dragon Isles (2025)

Route:
!9vvtZXjmm0FrTd2MpVM2lPhizc7m9MN9dmBO1b6WkstUS)2RLKbmBO5cJ)q6PNE6bOf6D6Q(J)YCcU4wc6Q)Am)wxgRRo1B7hE8G1aG5(UMEmYdJq)3WZB7oRV7MyQFtxQqiE9GD0GHFACyW0bpoAT6sX8(QXJwZRg(S)4UelDPGEeZlltrGi41v1hIAKPUJt8xkCxwke(D5UDU9Yv5NeKFwtttuuqgzy0ujaKYH5ftxPcYokcZpi7yoV4v5jO8KFgRJxXA5CHaKT7cUIVnHcxg0Y30pa2c7(ipv(uuRayv9YOdt3aBpRbuw3nfiXnmDbbbwcuOCnVJLOAMIHJlkOfIOnzBbcw(mFMoHMwtdKB6gGaB3sK8OlFtg7fWjzSGP8gCzPT4cl40JNRbSiDfZQjgzCWcX20vfmtM8afFKsZCq6N2iWP(kal8zRH0IKozUwIPmzLlsTyv)VmonedzigXZoXB7ivaKigzHymjrbiLo)(XmBuB3rKbrUQjyxhB6ypxcHiAaZirGCDKqJubvyeiY2kWZqqWZqMj4eLsotPItvgVT6qZovG6CRueVAqUQRxCRarGGoJfKmQTYP2QGAlK6PmZZyINZ8g9o(7LzCaYCFRvWHOI4wt5T2kPV1s3uQL88k6ZChW8xT)(y3ztFh9z(SmDfymVG)g4OU6IX6(bsBFxisdTnWdnnxm0FwijXp8bgxW8M75thARndF9(hUU)NUF9yF)6(N6hbZ19F56(7g7atNU6L6w8TQMbJ55EBnwHQJMZJTwKb4UPR(rFBNP2rkh01TnnTNgTW7oPanX6)9
Stacktraces

GetCurrentPreset:
[string "@Interface/AddOns/MythicDungeonTools/Modules/ErrorHandling.lua"]:261: in function `DrawHull'
[string "@Interface/AddOns/MythicDungeonTools/Modules/PullOutlines.lua"]:325: in function <...e/AddOns/MythicDungeonTools/Modules/PullOutlines.lua:315>

DrawHullFontString:
[string "@Interface/AddOns/MythicDungeonTools/Modules/ErrorHandling.lua"]:261: in function `DrawHullFontString'
[string "@Interface/AddOns/MythicDungeonTools/Modules/PullOutlines.lua"]:326: in function <...e/AddOns/MythicDungeonTools/Modules/PullOutlines.lua:315>

**Comment by Debugga-nb** [comment]
I've also shared the error to the MDT-Guide issues CurseForge.

**Comment by Nnoggie** [comment]
The ticket was closed because this issue is not an issue with MDT but with MDTGuide (a different AddOn that i have no control over).
Closed

**Comment by Nnoggie** [comment]
The previous issue was closed because the underlying issue in MDT that caused a bug that produced the same error message got fixed as outlined in the closing comment. The fix: https://github.com/Nnoggie/MythicDungeonTools/commit/f841f957f7babc674ab8b428b0edb4b38eda7199
Errors relating to MDTGuide cannot be fixed by me and need to be reported to MDTGuide authors.

**Comment by Nnoggie** [comment]
Additionally there was no new release yet after your issue was closed. A new release like https://github.com/Nnoggie/MythicDungeonTools/releases/tag/v4.0.4.25 is necessary to distribute changes. Please refrain from reopening / duplicating issues like this in the future.

**Comment by Debugga-nb** [comment]
> Additionally there was no new release yet after your issue was closed. A new release like https://github.com/Nnoggie/MythicDungeonTools/releases/tag/v4.0.4.25 is necessary to distribute changes. Please refrain from reopening / duplicating issues like this in the future.

I saw your update and closure, and simultaneously pulled the update that was flagged as available from Curse. Looks like it pulled the .24 version. The issue persisted, and admittedly I wasn't paying attention to sub-revisions, the next logical step is to report it.
Wasn't "trying" to duplicate. I've submitted a request with MDTGuide. 

Woah woah. I just pulled .25. So your answer is "make it not work at all"? I had a rough feeling coming here at all, as you have a history of being generally hostile. But oof. New lows.
